src/app/shell/details/details-tabs/reviews/reviews.component.ts

```diff
@@ -55,12 +59,17 @@ export class ReviewsComponent implements OnInit, OnDestroy {
   ngOnInit(): void {
     this.getParentData();
     this.getWorkshopRatingList();
+    this.checkIfAllowedToReview(this.parent)
     
     this.actions$.pipe(ofActionCompleted(OnCreateRatingSuccess))
       .pipe(
         takeUntil(this.destroy$),
         distinctUntilChanged())
       .subscribe(() => this.store.dispatch(new GetRateByEntityId('workshop', this.workshop.id)));
+
+    this.isAllowedToReview$
+    .pipe(takeUntil(this.destroy$))
+    .subscribe((status: boolean) => (this.isAllowedToReview = status));
   }
```

Comment:
remove please old functionality regarding receiving approved applications, because you are not using it anymore 
![image](https://user-images.githubusercontent.com/27493959/180783822-1902f479-99d8-4966-912c-c6de402c5fcb.png)